limitador-server/src/main.rs

```diff
@@ -552,6 +573,41 @@ fn create_config() -> (Configuration, String) {
     let matches = cmdline.get_matches();
 
     let limits_file = matches.value_of("LIMITS_FILE").unwrap();
+
+    if matches.is_present("validate") {
+        let error = match std::fs::File::open(limits_file) {
+            Ok(f) => {
+                let parsed_limits: Result<Vec<Limit>, _> = serde_yaml::from_reader(f);
+                match parsed_limits {
+                    Ok(limits) => {
+                        if limitador::limit::check_deprecated_syntax_usages_and_reset() {
```

Comment:
should `first_negative` be called to check existence of negative maxValues?

Reply:
Well it does return the first negative `max_value` as an `Option<usize>`

Reply:
`find_first_negative_limit` maybe?

Reply:
my point is, the validation command should also check there is no "negative limits", right?

Reply:
Oh! Yes!!!